tests/e2e/support/objects/app-files/spaces/actions.ts

```diff
@@ -206,3 +211,43 @@ export const openSpaceTrashBin = async (args: openSpaceTrashBinArgs): Promise<vo
   await page.locator(spaceContextButton).click()
   await page.locator(spaceDeletedFilesButton).click()
 }
+
+/**/
+
+export const changeSpaceImage = async (args: {
+  id: string
+  page: Page
+  resource: File
+}): Promise<void> => {
+  const { id, page, resource } = args
+  await sidebar.open({ page: page })
+  await sidebar.openPanel({ page: page, name: 'space-actions' })
+
+  const [fileChooser] = await Promise.all([
+    page.waitForEvent('filechooser'),
+    page.locator(editImageOptionSelector).click()
+  ])
+
+  await Promise.all([
+    page.waitForResponse(
+      (resp) =>
+        resp.url().endsWith(encodeURIComponent(id)) &&
+        resp.status() === 200 &&
+        resp.request().method() === 'PATCH'
+    ),
+    page.waitForResponse(
+      (resp) =>
+        resp.url().endsWith(resource.name) &&
+        resp.status() === 207 &&
+        resp.request().method() === 'PROPFIND'
+    ),
+    fileChooser.setFiles(resource.path)
+  ])
+
+  const img = await page.locator('.space-overview-image')
```

Comment:
Maybe we can store this selector in some variable like others

Reply:
You're right.  done